inventory-manager.lic

```diff
@@ -0,0 +1,125 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#invdb
+=end
+
+class InventoryManager
+  include DRC
```

Comment:
💡 Should this `custom_require` at least `common`? Maybe also `drinfomon` if that's where `Char.name` comes from.

Reply:
Yes, I am not sure how I forgot that.